src/avatar/avatar.js

```diff
@@ -38,8 +38,8 @@ export default class Avatar extends React.Component<PropsT, StateT> {
   }
 
   componentDidUpdate(prevProps: PropsT, prevState: StateT) {
-    if (prevProps.src !== this.props.src && prevState.noImageAvailable) {
-      this.setState({noImageAvailable: false});
+    if (prevProps.src != this.props.src) {
```

Comment:
is strict equality possible?

Reply:
yup it is possible. Initially i was thinking there is no need to setState if the `src` prop is updated from a null-ish to another null-ish value (e.g. null to undefined, and vice versa). But such scenario may never occur or not worth checking, so I can update this to strict equality